core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -1658,4 +1658,34 @@ class SQLPlanParserSuite extends BaseTestSuite {
     val pluginTypeChecker = new PluginTypeChecker()
     pluginTypeChecker.getNotSupportedExprs(filterExprArray) shouldBe 'empty
   }
+
+  runConditionalTest("WindowGroupLimitExec is supported", execsSupportedSparkGTE350) {
+    val windowGroupLimitExecCmd = "WindowGroupLimit"
```

Comment:
minor nit: variable `windowGroupLimitExecCmd` seems used only once, do we need it?

Reply:
Usually, I do not prefer having such strings as hard coded values. Updated the app name to use this variable as well.